readme.md

```diff
@@ -35,13 +35,13 @@ $ bitcoin-cli stop
 * get utreexo code
 
 ```
-$ go get github.com/mit-dci/utreexo
+$ go get github.com/chainsafe/utreexo
```

Comment:
Should change all of these back to mit-dci for a PR

Reply:
addressed